packages/loader/container-loader/closeAndGetPendingLocalState.md

```diff
@@ -0,0 +1,31 @@
+# Stashed Changes
+
```

Comment:
Who's the intended audience for this document? Since it's linked from a public API, I would've guessed that audience includes users of the API and not just implementers. Assuming that's correct, feels like an introductory sentence which orients the reader that this feature is related to offline might be nice. E.g.

"Pending local state is a mechanism that Fluid provides for mitigating data loss in certain offline scenarios."